_posts/2016-12-06-crystal-0.20.1-released.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Crystal 0.20.1 released!
+summary: Crystal has a new development release.
+thumbnail: +
+author: straight-shoota
+release_notes: true
+---
+
+[Crystal 0.20.1](https://github.com/crystal-lang/crystal/releases/tag/0.20.1) has been released!
+
+As any release it includes numerous bugfixes, cool features and performance improvements - in [75 commits since 0.20.0](https://github.com/crystal-lang/crystal/compare/0.20.0...0.20.1).
+
+# Exciting Changes
+
+* `p` and `pp` use pretty printing.
+* `finished` macro hook places it's contents at the end of the program when all other types and methods are known.
+* Local variables can now have type restrictions.
+* `method_missing` hook can now define a method, useful for specifiying block arguments.
```

Comment:
This is a bit ambiguous on what it means, a link to the issue, or an example, would be ideal.